roster2/src/handlebars/overview.handlebars

```diff
@@ -2,51 +2,50 @@
 	<span class="instruction rosterTitleMessage">{{translate 'title_msg'}}</span>
 {{/if}}
 
-<div class="roster-table">
-    <div class="roster-table-row">
-        <div class="roster-table-cell">{{translate 'groups'}}</div>
-        <div class="roster-table-cell">
-            <select id="roster-groups-selector-top">
-                <option value="all">{{translate 'roster_sections_all'}}</option>
-                {{#each siteGroups}}
-                <option id="roster-group-option-{{id}}" value="{{id}}">{{title}}</option>
-                {{/each}}
-            </select>
-        </div>
+<div class="row">
+    <div class="form-group col-sm-6">
+            <label for="roster-groups-selector-top" class="form-control-label col-sm-3">{{translate 'groups'}}</label>
+            <div class="col-sm-4">
+            	<select id="roster-groups-selector-top">
+                	<option value="all">{{translate 'roster_sections_all'}}</option>
+                	{{#each siteGroups}}
+                	<option id="roster-group-option-{{id}}" value="{{id}}">{{title}}</option>
+                	{{/each}}
+            	</select>
+            </div>
```

Comment:
The indentation looks a bit odd here. It looks like you've used hard tabs rather than spaces.